icu4j/main/classes/core/src/com/ibm/icu/impl/locale/XLikelySubtags.java

```diff
@@ -3,14 +3,6 @@
 package com.ibm.icu.impl.locale;
 
 import java.util.Collection;
```

Comment:
Why did you not move this line together with the others?

Also, Eclipse can or will auto-format imports in ICU4J. Does the new order conform to that?